rinex/src/qc/analysis/sv.rs

```diff
@@ -8,8 +8,7 @@ pub struct QcSvAnalysis {
 
 impl QcSvAnalysis {
     pub fn new(rnx: &Rinex, _nav: &Option<Rinex>, _opts: &QcOpts) -> Self {
-        let mut sv = rnx.space_vehicles();
-        sv.sort();
+        let sv = rnx.sv();
```

Comment:
I guess dropping the `sort` call was intended?

Reply:
I thought about this, and i'm not sure what the new behavior is: I did not take the time to re-run a QC report, I need to do that tonight. 

Having sorted (ordered) results is important to me and should be preserved, it only serves two cases:
  - when generating a RINEX file (`Rinex::to_file("test.rnx")`) we expose the satellite vehicles in sorted (alphabetically and increasing numbers), which I find much easier to navigate as far as i'm concerned. Cool thing about this lib, is you can use it to re-order an "unsorted" RINEX. Usually, modern RINEX are generated by modern tools that sort the data. And usually, older RINEX files (V2) are not sorted because they were generated by older tools
  - when generating QC reports, we order the Sv likewise, which is very pleasant to navigate

Reply:
I confirm QC report is nicely ordered